packages/salesforcedx-utils-vscode/src/helpers/index.ts

```diff
@@ -5,7 +5,7 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-export { isNullOrUndefined, extractJsonObject } from './utils';
+export { asyncFilter, isNullOrUndefined, extractJsonObject } from './utils';
```

Comment:
Added `asyncFilter` (which BTW, calling `map()` or `filter()` with async functions in them is a real PITA)